src/ai/ai.service.ts

```
@@ -49,4 +46,4 @@ ${diff}`;
       return 'Unable to analyze this PR due to an API error. Please check your Google API key and model configuration.';
     }
   }
-}
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)